FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/MaxAccumDuration.java

```diff
@@ -1,14 +1,16 @@
 package hec.ensemble.stats;
 
-import static hec.ensemble.stats.ConvertUnits.convertCfsAcreFeet;
+import javax.measure.IncommensurableException;
+import javax.measure.Unit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public  class MaxAccumDuration implements Computable, Configurable {
+    private static final Logger logger = Logger.getLogger(MaxAccumDuration.class.getName());
     private static final String DEFAULT_INPUT_UNITS = "cfs";
-    private static final long DEFAULT_INPUT_DURATION = 3600;  //seconds
-
     Integer accumulatingDuration; //duration in hours
     Configuration config;
-    private String outputUnit;
+    private Unit<?> unit = null;
```

Comment:
Is the null initializer of this global variable necessary?

Reply:
Removed initializer